src/app/QuickStarts/quickstarts/settings-quickstart.tsx

```diff
@@ -104,9 +104,9 @@ The **Automated Analysis** dashboard card allows you to automatically start an a
 2. Configure the **Credentials Storage** settings.
 
   Credentials are necessary to authenticate with the target JVMs that [APP] communicates with, if JMX auth is enabled or if a [APP] agent authenticated connection is required. If you prefer not to store these credentials in the [APP] backend, you can opt to store them in local session storage instead.
-3. Configure the **Feature Level** settings.
+3. Configure the **Feature level** settings.
```

Comment:
lowercase "feature level"?

Reply:
ooh no actually this is also cased to match the updated UI 
![image](https://github.com/cryostatio/cryostat-web/assets/56743004/715cefa4-f79b-431c-8266-b2158040079e)